packages/transformers/css/src/CSSTransformer.js

```diff
@@ -74,31 +73,35 @@ export default new Transformer({
         throw new Error('Could not find import name for ' + rule);
       }
 
-      if (PROTOCOL_RE.test(moduleSpecifier)) {
-        return;
+      if (isURL(moduleSpecifier)) {
+        name.value = asset.addURLDependency(moduleSpecifier);
```

Comment:
1. This is essentially a noop, but the HTMLTransformer does this as well (e.g. `<script src`) - it should at least be consistent.
2. Was there a reason for using `PROTOCOL_RE` instead of `isURL`?